src/Carbon/Carbon.php

```diff
@@ -1576,6 +1576,16 @@ public function isLeapYear()
     }
 
     /**
+     * Determines if the instance is in the current year
+     *
+     * @return bool
+     */
+    public function isCurrentYear()
+    {
+        return $this->year === static::now($this->tz)->year;
```

Comment:
I think you can directly call `format` as for `\Carbon\Carbon::isBirthday`: this avoids calling `\Carbon\Carbon::__get`